diff --git a/main.go b/main.go
index 043c407..a1685ab 100644
--- a/main.go
+++ b/main.go
@@ -4,6 +4,8 @@ import (
 	"context"
 	"log"
 	"log/syslog"
+	"net/http"
+	_ "net/http/pprof"
 
 	"golang.org/x/text/encoding/charmap"
 )
@@ -13,6 +15,9 @@ const (
 )
 
 func main() {
+	go func() {
+		log.Println(http.ListenAndServe("localhost:6060", nil))
+	}()
 	rootCtx, rootCancel := context.WithCancel(context.Background())
 	defer rootCancel()
 	logger, err := syslog.NewLogger(syslog.LOG_DEBUG, 0)
